sections/06_discussion.md

```diff
@@ -2,57 +2,78 @@
+to biomedical data compared to other domains where deep learning is applied.
+This experiment-level variation imposes limits on classification performance and
+can result in underestimated error metrics. To mitigate these issues and
+incorporate between-study variation into the prediction model itself, it is
+recommended to integrate data from multiple labs and experiments.
 
 #### Evaluation metrics for imbalanced classification
 
-Less than 1% of the genome can be confidently labeled as bound for most
-transcription factors. Therefore, it is important to evaluate the genome-wide
-recall and false discovery rate (FDR) of classification models of biochemical
-activities. Targeted validation experiments of specific biochemical activities
-usually necessitate an FDR of 5-25%. When predicted biochemical activities are
-used as features in other models, such as gene expression models, a low FDR may
-not be as critical if the downstream models can satisfy their evaluation
-criteria.  An FDR of 50% in this context may suffice.
-
-What is the correspondence between these metrics and commonly used
-classification metrics such as auPRC (area under the precision-recall curve) and
-auROC? auPRC evaluates the average precision, or equivalently, the average FDR
-across all recall thresholds. This metric provides an overall estimate of
-performance across all possible use cases, which can be misleading for targeted
-validation experiments. For example, classification of TF binding sites can
-exhibit a recall of 0% at 10% FDR and auPRC greater than 0.6. In this case, the
-auPRC may be competitive, but the predictions are ill-suited for targeted
-validation that can only examine a few of the highest-confidence predictions.
-Likewise, auROC evaluates the average recall across all false positive rate
-(FPR) thresholds, which is often a highly misleading metric in class-imbalanced
-domains [@doi:10.1145/1143844.1143874 @doi:10.1038/nmeth.3945]. For example,
-consider a classification model with recall of 0% at FDR less than 25% and 100%
-recall at FDR greater than 25%. In the context of TF binding predictions where
-only 1% of genomic regions are bound by the TF, this is equivalent to a recall
-of 100% for FPR greater than 0.33%. In other words, the auROC would be 0.9967,
-but the classifier would be useless for targeted validation. It is not unusual
-to obtain a chromosome-wide auROC greater than 0.99 for TF binding predictions
-but a recall of 0% at 10% FDR.
+Making predictions in the presence of high class imbalance and differences
+between training and generalization data is a common feature of many large
+biomedical datasets, including deep learning models of genomic features, patient
+classification, disease detection, and virtual screening.
+
+Prediction of transcription factor binding sites exemplifies difficulties with
+deep learning models of genomic features. The human genome has 3 billion base
```

Comment:
Perhaps `exemplifies the difficulties with learning from highly imbalanced data`?